modules/programs/librewolf.nix

```
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.librewolf;
+
+  mkOverrides = prefs: ''
```

Perhaps a bit clearer:
```suggestion
  mkOverridesFile = prefs: ''
```